.github/workflows/build_test.sh

```diff
@@ -121,7 +121,7 @@ for args in "${ARGS[@]}"; do
          CC="$CC" CC_LD="$LINKER" CFLAGS="-Werror" \
          CXX="$CXX" CXX_LD="$LINKER" CXXFLAGS="-Werror" \
          meson -Dtests=unsafe -Dslow-tests=true -Dfuzz-tests=true --werror \
-               $args build; then
+               -Dcryptolib="${CRYPTOLIB:?}" $args build; then
```

Comment:
The check whether this var is set should be at the top with the others ones (for early fail).

Reply:
Indeed, thanks.